test/itests/test-itests-common/src/test/java/ddf/common/test/KarafConsole.java

```diff
@@ -0,0 +1,66 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+package ddf.common.test;
+
+import java.security.Principal;
+
+import org.apache.karaf.itests.KarafTestSupport;
+import org.osgi.framework.BundleContext;
+
+/**
+ * Class that provides access to the Karaf Console.
+ *
+ * Note: This class is needed to expose the protected methods provided by {@link KarafTestSupport}.
+ * Since we already extend from our own base class, out test classes cannot extend from this one
+ * to access its protected methods.
+ */
+public class KarafConsole extends KarafTestSupport {
```

Comment:
@pklinef do you think it is worth a ticket? Otherwise I'm not sure how else to capture us revisiting this.